test/kernel_acl.js

```diff
@@ -103,6 +103,15 @@ contract('Kernel ACL', accounts => {
             const param = new web3.BigNumber(argId + op + value)
 
             const r1 = await acl.grantPermissionP(accounts[3], app, role, [param], { from: granted })
+
+            // retrieve the params back with the getters
+            const numParams = await acl.getPermissionParamsLength(accounts[3], app, role)
+            assert.equal(numParams, 1, 'There should be just 1 param')
+            const returnedParam = await acl.getPermissionParams(accounts[3], app, role, 0)
+            assert.equal(returnedParam[0].valueOf(), parseInt(argId), 'param id should match')
+            assert.equal(returnedParam[1].valueOf(), parseInt(op), 'param op should match')
+            assert.equal(returnedParam[2].valueOf(), parseInt(value), 'param value should match')
```

Comment:
We should be careful with the uses of `parseInt`, based on the string representations for each variable:

- `argId` should use `parseInt(argId, 16)`
- `op` should use `parseInt(op, 10)`
- `value` should use `parseInt(value, 10)`

Reply:
I don't think it would work with different encodings but it doesn't hurt (and it's probably clearer for hex case), so changing it.